go.mod

```diff
@@ -1,5 +1,5 @@
 module github.com/Scalingo/go-plugins-helpers
 
-go 1.15
+go 1.16
 
 replace github.com/Sirupsen/logrus v1.7.0 => github.com/sirupsen/logrus v1.7.0
```

Comment:
Why is this needed ?

Reply:
This dependency got renamed from `Sirupsen` to `sirupsen` a while back and it caused a mess in many Go projects around the world.

But you're right, we may not need it in this project. Let me check that.

The comprehensive answer is here: https://github.com/sirupsen/logrus/issues/570#issuecomment-313933276

Reply:
I removed it in b60c82e